pandas/core/window.py

```diff
@@ -602,8 +602,8 @@ def validate(self):
         if isinstance(window, (list, tuple, np.ndarray)):
             pass
         elif is_integer(window):
-            if window < 0:
-                raise ValueError("window must be non-negative")
+            if window <= 0:
+                raise ValueError("please enter a positive window")
```

Comment:
maybe : window must be > 0

Reply:
@jreback is this a suggestion for ```ValueError``` message?